.travis.yml

```diff
@@ -0,0 +1,62 @@
+language: python
+sudo: required
+dist: trusty
+env:
+  global:
+    -JYTHON=false
+before_install:
+  - if [ "$JYTHON" == "true" ]; then
+        hg clone http://hg.python.org/jython;
+        cd jython;
+        ant;
+        cd ..;
+        chmod +x "$(pwd)/jython/dist/bin/jython";
+    fi
+  - if [ "$JYTHON" == "true" ]; then
+        export PYTHON_EXE="$(pwd)/jython/dist/bin/jython";
+    else
+        export PYTHON_EXE=python;
+    fi
+  - if [ "$JYTHON" == "true" ]; then
+        alias jython="$(pwd)/jython/dist/bin/jython";
```

Comment:
Why all these separate `if [ "$JYTHON" == "true" ]`  sections? Why not combine to one? Or if not all, maybe at least this one could be combined with previous. Just an idea.

Reply:
Thank you for pointing that out! I blindly copy-pasted the jython configuration from here: https://github.com/PyCQA/astroid/commit/359118209ee5a458e83d5b137c2066447acdfd0d, hehe.